index.html

```diff
@@ -3554,14 +3554,15 @@ <h5>JWT Encoding</h5>
 in the <a>verifiable credential</a> <a>subject</a>.
               </li>
               <li>
-<code>aud</code> MUST represent the <code>subject</code> <a>property</a> of the
-consumer of the <a>verifiable presentation</a>.
+<code>aud</code> MUST represent the
+consumer of the <a>verifiable presentation</a> (i.e. the verifier).
               </li>
             </ul>
 
             <p>
 Other JOSE header parameters and <a>claim</a> names not specified herein can be
-used if their use is not explicitly discouraged. Additional <a>claims</a> MUST
+used if their use is not explicitly discouraged. Additional <a> verifiable 
+credential </a> <a>claims</a> MUST
```

Comment:
remove whitespace after `<a>` and before `</a>`.

Reply:
Done, thanks